app/views/static_pages/contact.html.erb

```diff
@@ -1,2 +1,17 @@
-<h1>StaticPages#contact</h1>
-<p>Find me in app/views/static_pages/contact.html.erb</p>
+<div class="content-wrap">
+  <div class="container footer-margin">
+    <div class="row">
+      <div class="col-sm-offset-2 col-sm-8 card content-top-margin">
+        <div class="row tutorial-min">
+          <div class="col-sm-12">
+            <h2>Contact Us</h2>
+            <hr />
+            <p>
+              If you have any comments, questions or concerns, or if you'd like to reach us in order to get more involved, please feel free to reach out to us at <a href="#">godigitalwithus@gmail.com</a>.
```

Comment:
Typically, the form will just fire off an email any way, but it decreases the amount of spam in two ways:

1. Captcha.  
2. No scrapable email address for bots just skimming the page.